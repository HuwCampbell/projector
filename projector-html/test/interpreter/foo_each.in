{ each [foo, foo, foo] text }