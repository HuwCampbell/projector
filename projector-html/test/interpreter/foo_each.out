Nested [ Plain "foo" , Plain "foo" , Plain "foo" ]